src/include/kernel/ext_manifest.h

```diff
@@ -112,4 +113,11 @@ struct ext_man_ipc_msg_size {
 	uint32_t ipc_msg_size;
 } __packed;
 
+/* CAVS LPRO default clock */
+struct ext_man_cavs_lpro_info {
+	struct ext_man_elem_header hdr;
+
+	uint32_t cavs_lpro;
+} __packed;
```

Comment:
This is going to grow quickly, I suggest an array of token/value here for IPC size too.

Reply:
ping. I dont want to have lots of single value structures. you could have a more generic structure whic could hold an array of 
```c
struct elem {
    uint32_t token;
    uint32_t value;
};
```
This could easily grow without having to ad new structures, we just add new tokens.

Reply:
Updated with your approach, please check.
Not sure about the names, let me know if you have something better than 'misc elem'